kura/org.eclipse.kura.core.cloud/src/main/java/org/eclipse/kura/core/cloud/CloudPayloadJsonDecoder.java

```diff
@@ -88,9 +88,15 @@ public static KuraPayload buildFromByteArray(byte[] array) {
     }
 
     private static void decodeTimestamp(KuraPayload payload, JsonValue timestampValue) {
-        if (timestampValue != null && timestampValue.isNumber()) {
-            long timestamp = timestampValue.asLong();
-            payload.setTimestamp(new Date(timestamp));
+        if (timestampValue != null) {
```

Comment:
Please apply the formatter and cleanup profiles to all your code changes to keep consistency.
You'll find them here: https://github.com/eclipse/kura/tree/develop/kura/setups/formatting

Reply:
Ok. This is my problem.